reference/forms/types/choice.rst

```diff
@@ -323,6 +324,14 @@ error_bubbling
 Set that error on this field must be attached to the field instead of
 the parent field (the form in most cases).
 
+trim
+~~~~
```

Comment:
I think we should move the description to its own file to be able to reuse it in other form types. This is currently missing in some documents (see the failing build).